### DISCUSSION TITLE
Outreach committee starting point!

### DISCUSSION THREAD

**Discussion by NicoleAbuhakmeh-zz** [dotnet-foundation/Home#17] [Outreach Committee]
The readme files of the group will point you in the direction of getting involved: https://github.com/dotnet-foundation/wg-outreach/blob/main/README.md

Check here for upcoming meeting agendas: https://github.com/dotnet-foundation/wg-outreach/tree/main/Meetings

**Comment by Programazing** [comment]
At the moment, the **Joining the Committee & Meetings** section has the following sentence: "To join the the committee and the meetings, _please request to [join the outreach team](https://github.com/orgs/dotnet-foundation/teams/outreach)_ and then post a message there with your desire to be added to the meeting invite."

The link leads to a disabled Discussions section.

Where should we be directing requests? I would absolutely love to join the committee.

**Reply by rprouse** [reply]
For now, add an issue to the repository for each committee. For outreach, https://github.com/dotnet-foundation/wg-outreach/issues

We'll figure out an easier way to do this soon.

**Reply by shawnwildermuth** [reply]
We're working on a comprehensive way to join the committee.
